Runtime/RoadNetwork/Structure/RnRoad.cs

```diff
@@ -757,5 +763,56 @@ public static LineCrossPointResult GetLaneCrossPoints(this RnRoad self, LineSegm
                 .Concat(self.SideWalks.SelectMany(s => s.SideWays));
             return RnEx.GetLineIntersections(lineSegment, targetLines);
         }
+
+        /// <summary>
+        /// selfの内部をlineが取っているとしたときに, selfの両端のWayとlineの最も近い点との距離を返す
+        /// </summary>
+        /// <param name="self"></param>
+        /// <param name="line"></param>
+        /// <param name="distance"></param>
+        /// <returns></returns>
+        public static bool TryGetNearestDistance(this RnRoad self, RnLineString line, out float distance)
+        {
+            distance = 0f;
```

Comment:
Min取ってるので常に0を格納してしまう？
```suggestion
            distance =  float.MaxValue;
```

Reply:
リファクタリング時のバグでした。
ちょっとこの形式と違いますが修正しました。